### ISSUE TITLE
Show generator version when running yo --generators

### ISSUE THREAD

**Issue by andersonkyle** [yeoman/yo#542]
## Type of issue

Feature request

----

When running `yo --generators` it would be helpful to display the version alongside the name of each generator.  This could either be the new default or could be selectively enabled with a verbose flag.  This would make it much easier to identify the versions of all currently installed generators.

By the looks of this [comment](https://github.com/yeoman/yo/issues/187#issuecomment-229382609), it appears this feature would be a welcome addition.

**Comment by SBoudrias** [comment]
I think that's a great idea! Would you be interested in sending us a PR?

**Comment by andersonkyle** [comment]
@SBoudrias Absolutely.  This feature requires a change to [yeoman-environment](https://github.com/yeoman/environment/pull/90) so that we have access to the generator version #.  If that PR looks good, I can finish up the work in this repo.

**Comment by eduardomrodrigues** [comment]
@andersonkyle, great ideia! Can I help you?
